dali/operators/image/crop/bbox_crop.cc

```diff
@@ -96,191 +96,232 @@ struct Range {
         "threshold_type",
-        R"code(Determines the meaning of ``thresholds``. By default refers to intersection-over-union (IoU) of
-the bounding boxes with respect to the cropping window. Alternatively, it could be set to ``overlap``
-to specify the fraction (by area) of the the bounding box which will fall inside the crop window
-(e.g. a threshold ``1.0`` means the whole bounding box must be contained in the resulting cropping window))code",
-        "iou")
+        R"code(Determines the meaning of ``thresholds``.
+
+By default, thresholds refers to the intersection-over-union (IoU) of the bounding boxes
+with respect to the cropping window. Alternatively, the threshold can be set to ``overlap`` to
+specify the fraction (by area) of the bounding box that will will fall inside the crop window.
+For example, a threshold value of ``1.0`` means the entire bounding box must be contained in the
+resulting cropping window.
+)code", "iou")
     .AddOptionalArg(
         "aspect_ratio",
-        R"code(Single or multiple valid aspect ratio ranges for cropping windows.
-
-For 2D bounding boxes, a single aspect ratio ``x/y`` range should be provided (i.e. ``[min_xy, max_xy]``)
-
-For 3D bounding boxes, three aspect ratio ranges are expected (``x/y``, ``x/z`` and ``y/z``) ranges, i.e
-``[min_xy, max_xy, min_xz, max_xz, min_yz, max_yz]``.
-Alternatively, if a single aspect ratio range is provided, the valid aspect ratio range will be the
-same for the three ratios.
+        R"code(Single or multiple valid aspect ratio ranges for the cropping windows.
 
-Value for ``min`` should be ``> 0.0`` and ``min <= max``.
+- Here some additional information: For 2D bounding boxes, one aspect ratio in the ``x/y`` range
+  should be provided (for example, ``[min_xy, max_xy]``).
+- | For 3D bounding boxes, the (``x/y``, ``x/z`` and ``y/z``) aspect ratio ranges are expected.
+  | An example is ``[min_xy, max_xy, min_xz, max_xz, min_yz, max_yz]``. Alternatively,
+    if an aspect ratio range is provided, the valid aspect ratio range will be the same for the
+    three ratios.
 
-By default, square windows are generated.
+The value for ``min`` should be greater than ``0.0``, and min should be less than or
+equal to the ``max`` value.  By default, square windows are generated.
 
-Note: Providing ``aspect_ratio`` and ``scaling`` is incompatible with specifying ``crop_shape``
-explicitly.)code",
+.. note::
+  Providing ``aspect_ratio`` and ``scaling`` is incompatible with explicitly
+  specifying ``crop_shape``.
+)code",
                     std::vector<float>{1.f, 1.f})
     .AddOptionalArg(
         "scaling",
-        R"code(Range ``[min, max]`` for crop size with respect to original image dimensions.
+        R"code(Range ``[min, max]`` for the crop size with respect to the original image dimensions.
 
-Value for ``min`` should satisfy ``0.0 <= min <= max``.
+The value for min should be greater than ``0.0``, and min should be less than or
+equal to the max value.
```

Comment:
Again, we're not reading it aloud - the original inequality was much easier to understand.

Reply:
Done